tests/test_google_analytics_start_date.py

```diff
@@ -81,6 +81,7 @@ def test_run(self):
                 expected_start_date_2 = self.start_date_2
 
                 # collect information for assertions from syncs 1 & 2 base on expected values
+                # Exclude the last 2 days to avaoid instability in the tests- TDL-22494
                 record_count_sync_1 = record_count_by_stream_1.get(stream, 0)
```

Comment:
I'd suggest only making the call to pendulum.now() one time here and then using that value below like:
golden_date = pendulum.now().subtract(days=1)  # and I'd like to strip the hours, mins, seconds so it goes back to midnight two days ago.  Then in the if statements below we can say pedulum.parse(message_date) < golden_date

Reply:
Made it a variable and comparing. 
Changed pendulum.now() to pendulum.today() to take care of the hours, mins and seconds to be 0s